week-15/quizzes/03-reducers.md

```diff
@@ -0,0 +1,92 @@
+# Reducers Quiz
+
+This quiz is to test you on what reducers are and how to use them.
+
+## The questions
+
+<quiz>
+  <question>
+    <p>
+      The purpose of a reducer function is to:
+    </p>
+    <answer>Reduce the `state` object to a single value</answer>
+    <answer correct>Update `state`</answer>
+    <answer>Make a new `state` object</answer>
+    <answer>Dispatch an `action`</answer>
+    <explanation>
+      The Redux store has a reducer function for updating the current state and 
+      returning that new state. 
+    </explanation>
+  </question>
+</quiz>
+<quiz>
+  <question multiple>
+    <p>
+      A reducer accepts which of the following parameters? 
+    </p>
+    <answer>Initial `state`</answer>
+    <answer>Payload</answer>
+    <answer correct>Current `state`</answer>
+    <answer>Previous `state`</answer>
+    <answer>Dispatch</answer>
+    <answer correct>Action</answer>
+    <explanation>
+      The reducer function receives the current state and an action, updates the
+      state appropriately based on the action.type. 
```

Comment:
"The reducer function receives the current state and and action, [then] updates the state appropriately based on the `action.type`."

Added "then" after the comma and the backticks around the `action.type`.